packages/advanced-logic/specs/payment-network-erc20-address-based-0.1.0.md

```diff
@@ -29,6 +21,11 @@ As a payment network, this extension allows to deduce a payment `balance` for th
 
 Note: to use the Rinkeby testnet just replace the id by "pn-rinkeby-erc20-address-based"
 
+## Interpretation
+
+Any transaction from the request `currency` ERC-20 contract destined to the `addPaymentAddress` is considered as a payment.
+Any ERC-20 transaction reaching the address `addRefundAddress` is considered as a refund. 
```

Comment:
Since we're changing this, we also need to add that its any ERC-20 transaction from the request `currency`.

Reply:
same as above, the name of the address should be `paymentAddress`/`refundAddress`